pmm-app/tests/verifyPMMSettingsPageFunctionality_test.js

```diff
@@ -0,0 +1,56 @@
+Feature('PMM Settings Page Functionality');
+
+Before((I, loginPage, pmmSettingsPage) => {
+    I.amOnPage(loginPage.url);
+    loginPage.login("admin", "admin");
+    I.amOnPage(pmmSettingsPage.url);
+});
+
+Scenario('Open PMM Settings page and verify changing Metrics Resolution', async (I, pmmSettingsPage) =>{
+    let resolutionToApply = "Low";
+    pmmSettingsPage.waitForPmmSettingsPageLoaded();
+    await pmmSettingsPage.selectMetricsResolution(resolutionToApply);
+    await pmmSettingsPage.verifySuccessfulPopUp(pmmSettingsPage.messages.successPopUpMessage);
+    await pmmSettingsPage.verifyResolutionIsApplied(resolutionToApply);
+});
+
+xScenario('Open PMM Settings page and verify changing Data Retention', async (I, pmmSettingsPage) =>{
+    let dataRetentionValue = "1";
+    pmmSettingsPage.waitForPmmSettingsPageLoaded();
+    pmmSettingsPage.changeDataRetentionValueTo(dataRetentionValue);
+    await pmmSettingsPage.verifySuccessfulPopUp(pmmSettingsPage.messages.successPopUpMessage);
+    await pmmSettingsPage.verifyDataRetentionValueApplied(dataRetentionValue);
+});
+
+Scenario('Open PMM Settings page and verify adding Alertmanager Rule', async (I, pmmSettingsPage) =>{
+    let scheme = "http://";
+    let sectionNameToExpand = "Alertmanager integration";
```

Comment:
Both `scheme` and `sectionNameToExpand` should be constants.
We should try to apply this rule of thumb universally:  if a variable does not need to be mutated, it should be a constant.